### ISSUE TITLE
Rule Change: padded-blocks

### ISSUE THREAD

**Issue by bezenson** [eslint/eslint#16005]
### What rule do you want to change?

padded-blocks

### What change to do you want to make?

Generate more warnings

### How do you think the change should be implemented?

A new option

### Example code

```js
const a = {
    a: 1,

};
```

### What does the rule currently do for this code?

Nothing

### What will the rule do after it's changed?

Remove empty line

```
const a = {
    a: 1,
};
```

### Participation

- [ ] I am willing to submit a pull request to implement this change.

### Additional comments

Currently padded-blocks works on classes, switches, if-else. But it is not working on objects. Maybe it will be good to add this option?
